README.md

```diff
@@ -296,15 +296,15 @@ For more info about compliance requirements at FINOS, checkout our [Contribution
 
 ## Roadmap
 1. ~~Add documentation into [community.finos.org](community.finos.org)~~
-2. ~~Publish post on FINOS blog~~ - https://www.finos.org/blog/introducing-finos-security-scanning
+2. ~~Publish post on FINOS blog~~ - https://www.finos.org/blog/introducing-finos-code-scanning
```

Comment:
@TheJuanAndOnly99 - please revert this change, the blogpost url should not be changed.

Reply:
reverted